### ISSUE TITLE
very hard to get deep chem to work on conda enviroment. 

### ISSUE THREAD

**Issue by TheStoneMX** [deepchem/deepchem#1648]
**Description:**

- DeepChem Version:
- Platform: Ubuntu-Mint

It has been a real challenge to get this to work, besides the bugs... and still trying.... I write these lines to other people that are trying to get it to work

1.- conda Linux 4.7 has problems and cant install many packages, you have to downgrade to conda 4.6 ( In my case it was deepchem-2.2.0-py37_0.tar.bz2 )

2.- first you need to install conda 4.7 then run this command to downgrade.
conda install conda-4.6.14-py37_0.tar.bz2

3.- then you need to isntall these packages in order.
conda install -c rdkit rdkit
conda install -c conda-forge opencv
conda install -c conda-forge joblib

4.- in my case I isntalled keras-gpu which installs keras and tensorflow-gpu
conda install -c anaconda keras-gpu

5.- then none of the link in https://anaconda.org/deepchem/ works
conda install -c deepchem deepchem-gpu
Collecting package metadata: done
Solving environment: failed
PackagesNotFoundError: The following packages are not available from current channels:
------------------------------------
conda install -c deepchem deepchem
Collecting package metadata: done
Solving environment: failed
PackagesNotFoundError: The following packages are not available from current channels:

6.- does not work either....
conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.1.0

7.- conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.2.0 doesn't work for me either !!!!!! this is a real pain in the .....

8.- The only way I got it to work is downaloading this file conda-4.6.14-py37_0.tar.bz2 from https://anaconda.org/deepchem/deepchem/files with this command.
conda install deepchem-2.2.0-py37_0.tar.

the only problem with this approach is that once you do this you break conda environment and cant install many other things.....

9.- I will try to install from source and see.... before I break the conda environment again. I had to install conda like 6 times..... did not worked either...

**Comment by jordenkass** [comment]
Try the installation via the guide I wrote

https://github.com/jordenkass/deepchem-for-dummies

**Comment by peastman** [comment]
It works fine for me.  I'm on Ubuntu 16.04.  I first verified I had conda 4.7:

```
$ conda --version
conda 4.7.10
```

I created a clean environment to install into:

```
conda create --name test
conda activate test
```

And then installed with the same command you used:

```
conda install -c deepchem -c rdkit -c conda-forge -c omnia deepchem=2.1.0
```

Everything installed without problem.  I verified I could `import deepchem` and didn't get any errors.

Perhaps you're running into version conflicts with other libraries in your environment?  Try creating a clean environment and see if it works then.
